server/main.go

```diff
@@ -38,6 +38,10 @@ func getWebHandler(activeFileManager *ActiveFileManager, fileStore FileStore) ht
 		method := req.Method
 		path := urlPathToArray(req.URL.Path)
 
+		res.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
> The difference between an app client doing this and a browser is that running an app client requires the user to authenticate the app before it can make the request. With the browser, the user may be looking at pictures of cats, not knowing that their InstantShare session is being hijacked.

Can you elaborate on this? I don't understand what you mean.

How does the app client require user to authenticate?

What is an InstantShare session and how can it be hijacked?